tests/statistics_test.sh

```diff
@@ -194,11 +196,39 @@ function month_statistics_Test
 function year_statistics_Test
 {
   local target_year="2019"
+  local year_data
   local msg="Currently, kw does not have any data for the requested year."
+  local line
 
-  ID=1
+  declare -a expected_cmd=(
+    "2020 summary"
+    "Total   Max  Min       Average"
+    "Build   19   00:03:40  00:00:06  00:00:23"
+    "Deploy  8    00:01:13  00:00:31  00:01:04"
+  )
+
+  setUp
+
+  year_data=$(year_statistics "$target_year")
+  assertEquals "($LINENO)" "$msg" "$year_data"
+
+  target_year="2020"
+  year_data=$(year_statistics "$target_year")
+  compare_command_sequence expected_cmd[@] "$year_data" "$LINENO"
+
+  declare -a expected_cmd=(
+    "2021 summary"
+    "Total          Max  Min       Average"
+    "Uninstall      3    00:00:03  00:00:02  00:00:02"
+    "Build_failure  1    00:00:01  00:00:01  00:00:01"
+    "Build          3    00:00:55  00:00:01  00:00:19"
+    "Deploy         1    00:00:21  00:00:21  00:00:21"
+    "List           2    00:00:08  00:00:00  00:00:04"
+  )
+
+  target_year="2021"
```

Comment:
You don't need to reuse this variable here; put 2021 directly in the `year_statistics`.